apps/website/src/components/utils/Footer.tsx

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+export default (props: React.ComponentProps<'footer'>) => {
+  const currentYear = new Date().getFullYear();
+  return <footer {...props}>© {currentYear} Bentley Systems, Inc</footer>;
```

Comment:
does it need to be a react component?

Reply:
Do you want it in Astro or inline?

I did react because it's easy to include variables in html.

Reply:
well, we want components to be astro by default (just as easy to include variables in html), and only use react when astro doesn't work

Reply:
Changed to Astro.